xapian-applications/omega/configure.ac

```diff
@@ -261,6 +261,17 @@ PKG_CHECK_MODULES([POPPLER], [poppler-cpp], [
                      OMINDEX_MODULES="$OMINDEX_MODULES omindex_pdf"])
      LIBS=$save_LIBS],[ ])
 
+dnl check if libe-book is available.
+PKG_CHECK_MODULES([LIBEBOOK], [libe-book-0.1,librevenge-0.0,librevenge-generators-0.0,librevenge-stream-0.0], [
+     save_LIBS=$LIBS
+     LIBS="$LIBS $LIBEBOOK_CFLAGS $LIBEBOOK_LIBS"
+     AC_CHECK_HEADERS([librevenge-0.0/librevenge-generators/librevenge-generators.h \
```

Comment:
Is it okay to add LIBEBOOK_CFLAGS to LIBS?

Reply:
No, `CFLAGS` are used for compiling, `LIBS` for linking.